examples/embed/example_vertexai.py

```diff
@@ -0,0 +1,24 @@
+from unstructured.documents.elements import Text
+from unstructured.embed.vertexai import VertexAIEmbeddingConfig, VertextAIEmbeddingEncoder
+
+# https://python.langchain.com/docs/integrations/text_embedding/google_vertex_ai_palm
+# To use Vertex AI PaLM you must either have credentials configured for your environment (gcloud,
+# workload identity, etc…), or store the path to a service account JSON file as the
+# GOOGLE_APPLICATION_CREDENTIALS environment variable.
+
+# Or, you can pass the json content of your API key to the VertexAIEmbeddingConfig
+# like this: VertexAIEmbeddingConfig(api_key_json=GOOGLE_APPLICATION_CREDENTIALS_JSON_CONTENT)
+# this will create a file in the working directory with the content of the json, and set the
+# GOOGLE_APPLICATION_CREDENTIALS environment variable to the path of the file.
+
+embedding_encoder = VertextAIEmbeddingEncoder(config=VertexAIEmbeddingConfig())
```

Comment:
Vertext

Reply:
I think this code should just be the exact same as the documentation code. So:
needs 
`import os`
and `(api_key=os.environ["GOOGLE_APPLICATION_CREDENTIALS"])`

Reply:
[working example in examples with comments updated](https://github.com/Unstructured-IO/unstructured/pull/2693/commits/5b278bf93fe0fd08eb0a97ed66485f049c960d6c)